src/app/routes/index.js

```diff
@@ -6,6 +6,7 @@ import mostRead from './mostRead';
 import onDemandRadio from './onDemandRadio';
 import error from './error';
 import errorNoRouteMatch from './errorNoRouteMatch';
+import index from './idx';
```

Comment:
Was getting errors renaming the folder to `index` so I've left it as `idx`

Reply:
🤔 Would it be worth renaming `index` to `idx` just for clarity?
In most cases, the name`index` indicates that it's the default page/script

Reply:
Clashes with 'index' is a fun one! `idx` may be better because of this!

Reply:
This has been done now:)